src/lib.rs

```diff
@@ -237,6 +237,13 @@ pub struct AtomicPtr<T, F = domain::Global, P = alloc::boxed::Box<T>>(
     PhantomData<(F, *mut P)>,
 );
 
+#[cfg(all(not(loom), feature = "std"))]
```

Comment:
I think you can get rid of this line if you change `std::` to `core::` below, no?

Reply:
I think I still need the `not(loom)` as it seemed loom did not implement `Debug`, but let me check again.

Oh, it seems loom fell into the `derive` issue for `derive(Debug)` on a struct with a type parameter since it only conditionally implements `Debug` if the type parameter does. `std::sync::atomic::AtomicPtr` is implemented to not care about the type param.

Reply:
That seems worth fixing in loom itself!

Reply:
Fix for loom: https://github.com/tokio-rs/loom/pull/255